recipes/simbody/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "3.5.3" %}
+
+package:
+    name: simbody
+    version: {{ version }}
+
+source:
+    fn: Simbody-{{ version }}.tar.gz
+    url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
+    sha256: 8005fbdb16c6475f98e13b8f1423b0e9951c193681c2b0d19ae5b711d7e24ec1
+
+build:
+    number: 0
+    skip: true    # [win and py35]
+    features:
+        - vc14    # [win and py35]
+
+# NOTE: This installs the minimum dependencies for Simbody, i.e., no
+# visualizer. I'm also not completely sure if blas and lapack are necessary for
+# the unix builds because they are supposedly included with Simbody. Once
+# FreeGLUT, Xi, and Xmu are avaliable via Anaconda.org (free or conda-forge)
+# they can be added.
+requirements:
+    build:
+        - toolchain  # [unix]
+        - cmake
+        - blas       # [not win]
+        - lapack     # [not win]
```

Comment:
Since we are using a blas this needs to change a bit. Please mimic what is done in this [`meta.yaml`]( https://github.com/conda-forge/numpy-feedstock/blob/c01758dac8f95d3696da0031f422574656beb483/recipe/meta.yaml ).